src/LevelManager.cc

```diff
@@ -111,6 +111,34 @@ void LevelManager::ReadLevelPerformerInfo()
   this->runner->entityCompMgr.CreateComponent(this->worldEntity,
       components::Physics(*physics));
 
+  // Populate physics options that aren't accessible outside the Element()
+  // See https://github.com/osrf/sdformat/issues/508
+  if (physics->Element())
+  {
+    if (physics->Element()->HasElement("dart"))
```

Comment:
```suggestion
  if (physics->Element() && physics->Element()->HasElement("dart"))
```
Nit: I believe this can be condensed into a single `if`.